engine/dlib/src/dlib/http_cache.h

```diff
@@ -142,6 +142,8 @@ namespace dmHttpCache
      */
     Result Begin(HCache cache, const char* uri, uint32_t max_age, HCacheCreator* cache_creator);
 
```

Comment:
I would argue that these old Begin() functions aren't needed anymore?
If not, i propose we remove them.

Reply:
Removed!